pages/project/[id].js

```diff
@@ -20,52 +32,104 @@ export default function Task({
-                <input
-                  type="checkbox"
-                  checked={task.isDone}
-                  onChange={() => handleTaskCheckbox(task.id, id)}
-                />
-                {task.task}
-              </label>
-              <DeleteButton
-                type="button"
-                onClick={() => {
-                  handleDeleteTask(task.id, id);
-                }}
-              >
-                x
-              </DeleteButton>
-            </TaskItem>
-          ))}
-        </StyledList>
-      </Wrapper>
-      <TaskForm onHandleAddTask={onHandleAddTask} projectid={id} />
+      <Header>
+        <BackButton>
+          <Link href="/"> back </Link>
+        </BackButton>
+        <Title>{selectedProject.title}</Title>
+      </Header>
+      <Body>
+        <TaskWrapper>
+          <StyledList>
+            {selectedProject.tasks.map((task) => (
+              <TaskItem key={task.id}>
+                <label>
+                  <input
+                    type="checkbox"
+                    checked={task.isDone}
+                    onChange={() => handleTaskCheckbox(task.id, id)}
+                  />
+                  {task.task}
+                </label>
+                <DeleteButton
+                  type="button"
+                  onClick={() => {
+                    handleDeleteTask(task.id, id);
+                  }}
+                >
+                  x
+                </DeleteButton>
+              </TaskItem>
+            ))}
+          </StyledList>
+        </TaskWrapper>
+        <NotesForm onSubmit={handleSubmit}>
+          <Textarea
+            rows={10}
+            maxLength="250"
+            placeholder="Write your note here .."
+            name="note"
+            defaultValue={selectedProject.notes}
+            onChange={(event) => setCount(event.target.value.length)}
+          ></Textarea>
+          {count < 250 && count >= 0 ? (
+            <NotesCounter>{250 - count}</NotesCounter>
+          ) : (
+            <NotesCounter>0</NotesCounter>
+          )}
+          <SaveNoteButton type="submit">save note</SaveNoteButton>
+        </NotesForm>
+        <TaskForm onHandleAddTask={onHandleAddTask} projectid={id} />
+      </Body>
     </>
   );
 }
 
-const Wrapper = styled.div`
-  margin: 0 0 55px 0;
+const Header = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 0.9em;
+  align-items: center;
+`;
+const BackButton = styled.div`
+  position: absolute;
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+  left: 1em;
+  width: 3.3em;
+  outline: 0;
+  cursor: pointer;
+  border: none;
+  line-height: 2em;
+  border-radius: 7px;
+  font-weight: 400;
+  background: #9fa3ac;
+  color: #494b49;
+  box-shadow: 0 4px 14px 0 rgb(0 0 0 / 10%);
 `;
 
 const Title = styled.h1`
+  position: absolute;
+  top: 2em;
   font-style: bold;
   font-size: 17px;
+  color: #494b49;
 `;
 
-const BackButton = styled.div`
-  margin: 5px;
+const Body = styled.div`
```

Comment:
Body as name is misleading as you do not use a body element here. This could be a main element and, therefore, the `Main` could be a good name.